README.md

```diff
@@ -0,0 +1,734 @@
+
+## Access control
+---
+
+All requests support below authorization methods:
+ * Basic authorization with user and password. (
+  For both standard, enterprise and lite plans, user is 'token', password is the API key from `ibmcloud resource service-keys` for the service instance.)
+ * Bearer authorization with bearer token. (This token can be either API key or JWT token obtained from IAM upon login to IBM Cloud. Use `ibmcloud iam oauth-tokens` to retrieve the token after `ibmcloud login`)
+ * `X-Auth-Token` header to be set to the API key. This header is deprecated.
+
+##  Administration API endpoint
+---
+Administration API endpoint is the `kafka_admin_url` property in the service key for the service instance. This command can be used to retrieve this property.
+```bash
+$ibmcloud resource service-key "${service_instance_key_name}" --output json > jq -r '.[]|.credentials.kafka_admin_url'
+```
+
+## Environment Setup
+In the examples you must set and export environment variables as follows:
+- Either the `API_KEY` or `BEARER_TOKEN` to use for authentication.
+- `KAFKA_ADMIN_URL` to point to your Eventstreams admin endpoint.
+
+In addition, the `Content-type` header has to be set to `application/json`.
+
+Common HTTP status codes:
+- 200: Request succeeded.
+- 202: Request was accepted.
+- 400: Invalid request JSON.
+- 401: The authentication header is not set or provided information is not valid.
+- 403: Not authorized to perform the operation. Usually it means the API key used is missing a certain role. More details on what role can perform what operation refers to this [document](https://cloud.ibm.com/docs/services/EventStreams?topic=eventstreams-security).
+- 404: Unable to find the topic with topic name given by user.
+- 422: Semantically invalid request.
+- 503: An error occurred handling the request.
+
+Error responses carry a JSON body like the following:
+```json
+{"error_code":50301,"message":"Unknown Kafka Error", "incident_id": "17afe715-0ff5-4c49-9acc-a4204244a331"}
+```
+Error codes are of the format `HHHKK` where `HHH` is the HTTP Status Code and `KK` is the Kafka protocol error.  
+
+For E2E debugging purposes, the transaction ID of every request is returned in the HTTP header `X-Global-Transaction-Id`.
+If the header is set on the request, it will be honored. If not, it will be generated.
+In the event of a non-200 error return code, the transaction ID is also returned in the JSON error response as `incident_id`.
+
+
+## Using the REST API to administer Kafka topics
+---
+
+
+To run the example :-
+
+Compile the code.
+```sh
+    cd modules/example
+	mvn clean install
+```
+Set the required environment variables
+```sh
+# Set you API KEY.
+export API_KEY="abc123456789"
+
+# Set the Admin Endpoint to point to your cluster.
+export KAFKA_ADMIN_URL="https://xyzclustername.svc01.region.eventstreams.test.cloud.ibm.com"
+
+```
+
+Run the example
+```sh
+java -jar target/adminrest-examples-99-SNAPSHOT
```

Comment:
This line needs .jar at the end e.g. java -jar target/adminrest-examples-99-SNAPSHOT.jar

Reply:
fixed in https://github.ibm.com/mhub/admin-rest-sdk/pull/50